lib/ash/actions/create/create.ex

```diff
@@ -21,6 +21,7 @@ defmodule Ash.Actions.Create do
     else
       {changeset, opts} = Ash.Actions.Helpers.set_context_and_get_opts(domain, changeset, opts)
       changeset = Helpers.apply_opts_load(changeset, opts)
+      changeset = Ash.Notifier.PubSub.Helpers.add_topic_generator_loads(changeset)
```

Comment:
We shouldn't have behavior for a specific notifier here, it should be calling a callback on all of the notifiers for this resource and action.